README.md

```diff
@@ -169,11 +168,9 @@ Botkit is part of Microsoft Bot Framework and is released under the [MIT Open So
 [106]:https://github.com/howdyai/botkit/tree/master/packages/botbuilder-adapter-twilio-sms#readme
 [107]:https://github.com/howdyai/botkit/tree/master/packages/botbuilder-adapter-web#readme
 
-### Bot Framework Solutions (Preview)
+### Bot Framework Virtual Assistant
```

Comment:
I think we should title this "Bot Framework Virtual Assistant Solution Accelerator".  We need to start the process of differentiating the solution effort from the service effort.

Reply:
Amended.